zebedee-reader/Dockerfile.local

```diff
@@ -18,5 +18,7 @@ ENTRYPOINT java $JAVA_OPTS \
      -Dcontent_dir=$CONTENT_DIR \
      -DSTART_EMBEDDED_SERVER=N \
      -Drestolino.packageprefix=$PACKAGE_PREFIX \
+     -javaagent:zebedee-cms/target/dependency/aws-opentelemetry-agent-1.30.0.jar \
```

Comment:
Should this be aws-opentelemetry-agent-1.31.0.jar for consistency with the babbage version?